Client/Application/AppDelegate.swift

```diff
@@ -131,6 +133,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         
         adjustIntegration = AdjustIntegration(profile: profile)
 
+        // We need to check if the app is a clean install to use for
+        // preventing the What's New URL from appearing.
+        if getProfile(application).prefs.stringForKey(LatestAppVersionProfileKey) == nil {
```

Comment:
This is not good :-( What this means is that if you upgrade from 1.4, which does not have that pref set, it will set it to 2.0. That means that while we should show the What's New page, we won't, because the test we do in BVC, comparing the current version to the last version seen, will never pass.